contributors/mentoring/draft-kubernetes-pilot-mentor-guide.md

```diff
@@ -0,0 +1,370 @@
+---
+kep-number: draft-20172411
+title: My First KEP
```

Comment:
What is a KEP?

Reply:
FYI: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/1-kubernetes-enhancement-proposal-process.md

Reply:
Typo ;)